www/squid/src/opnsense/mvc/app/models/OPNsense/Proxy/Proxy.xml

```diff
@@ -263,6 +263,13 @@
                             <field1>sslurlonly</field1>
                         </addFields>
                     </check001>
+                    <check002>
+                        <ValidationMessage>When enabling "Record TLS connection pre-master secret and encryption details", SSL inspection must also be enabled</ValidationMessage>
```

Comment:
Isn't this the other way around? When `tlskeylog` is enabled, `sslbump` should also be configured?